test/Test/Database/LSMTree/Normal/StateMachine.hs

```diff
@@ -383,37 +379,20 @@ createSystemTempDirectory prefix = do
   Key and value types
 -------------------------------------------------------------------------------}
 
--- TODO: maybe use reference impl generators here?
-
-newtype Key1   = Key1   { _unKey1 :: QC.Small Word64 }
+newtype Key = Key KeyForIndexCompact
   deriving stock (Show, Eq, Ord)
   deriving newtype (Arbitrary, R.SerialiseKey)
-newtype Value1 = Value1 { _unValue1 :: QC.Small Word64 }
 
-  deriving stock (Show, Eq, Ord)
+newtype Value = Value SerialisedValue
+  deriving stock (Show, Eq)
   deriving newtype (Arbitrary, R.SerialiseValue)
-newtype Blob1  = Blob1  { _unBlob1 :: QC.Small Word64 }
 
-  deriving stock (Show, Eq, Ord)
-  deriving newtype (Arbitrary, R.SerialiseValue)
-
-instance R.Labellable (Key1, Value1, Blob1) where
-  makeSnapshotLabel _ = "Key1 Value1 Blob1"
-
-newtype Key2   = Key2   { _unKey2   :: KeyForIndexCompact }
-  deriving stock (Show, Eq, Ord)
-  deriving newtype (Arbitrary, R.SerialiseKey)
-
-newtype Value2 = Value2 { _unValue2 :: BS.ByteString }
-  deriving stock (Show, Eq, Ord)
-  deriving newtype (Arbitrary, R.SerialiseValue)
-
-newtype Blob2  = Blob2  { _unBlob2  :: BS.ByteString }
-  deriving stock (Show, Eq, Ord)
+newtype Blob = Blob LargeRawBytes
+  deriving stock (Show, Eq)
```

Comment:
For tests, I don't think it's so important that the blob data is actually large. And it might speed up the tests a bit if we use some small type

Reply:
That's true. We get interesting behaviour for larger values (overflow pages and index fun), but not large blobs.